app/scripts/models/application-settings.coffee

```diff
@@ -273,6 +285,7 @@ define [
         includeInJSON: 'id'
     ]
 
+    version: 'da'
 
```

Comment:
Yeah, you're doing it right.

For how to declare class and instance methods in coffeescript, see: http://coffeescript-cookbook.github.io/chapters/classes_and_objects/class-methods-and-instance-methods